src/components/utils/file.js

```diff
@@ -1,4 +1,4 @@
-import loadImage from 'blueimp-load-image/js/load-image'
+import loadImage from 'blueimp-load-image/js'
```

Comment:
I see. So the other modules required in the index are also necessary for what we want, I guess?